molecule/validators.py

```diff
@@ -35,14 +35,19 @@ def check_trailing_cruft(ignore_paths=[]):
     filenames = []
     pruned_filenames = []
     found_error = False
+    valid_extensions = ['py', 'yml', 'rb']
```

Comment:
Thought about it, but not really sure it mattes.  Was mostly trying to keep the config from growing excessively, and couldn't think of a good reason people would be changing this.  I assume we can make it a config option if it displeases people ;)  This was me asserting some conventions w/o the configuration :)